ApeWorX/ape
# fix: fixed an extra parenthesis in pk export

### What I did

removed an extra parenthesis in the display of the private key when exporting account private key.

### How I did it

removed extra character from f-string

### How to verify it

type in console

```
ape accounts export <alias>
```
and verify that the sk does not contain an extra parenthesis at the end
